### PR TITLE
chore(doc): add TRG 5.06 - Application Configuration

### PR DESCRIPTION
**Pull request by carslen**
Introduce new TRG 5.06 Application configuration

### CODE REVIEW

File: docs/release/trg-5/trg-5-6.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+title: TRG 5.06 - Application Configuration
+---
+
+| Author               | Status | Created     | Post-History    |
+|----------------------|--------|-------------|-----------------|
+| Catena-X System Team | Active | 30-Nov-2022 | Initial release |
+
+## Description
+
+Every configuration aspect of your application has to be configurable for your Helm Chart.
```
Comment (Siegfriedk) [inline docs/release/trg-5/trg-5-6.md:11]: Perhaps we can say "has to be configurable through your helm chart"?

File: docs/release/trg-5/trg-5-6.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Why
+
+The application will be deployed in different environments and application operation must be able to adjust the
```
Comment (Siegfriedk) [inline docs/release/trg-5/trg-5-6.md:15]: I would rename "application operation" into something else; perhaps users? or operators?

File: docs/release/trg-5/trg-5-6.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+deployment to environmental needs.
+
+Examples for configurable items:
+
+- Ingress class, URLs, TLS configuration, etc.
+- Resource annotations and labels
+- Signing keys
+- Database connection strings
+- Secrets (with empty values!)
+- Persistent Storage Claims
```
Comment (Siegfriedk) [inline docs/release/trg-5/trg-5-6.md:25]: I think its the storage class of a pvc? Its also persistent volumne claim

File: docs/release/trg-5/trg-5-6.md (hunk 1 above)
Comment (carslen) [reply docs/release/trg-5/trg-5-6.md]: Check.

File: docs/release/trg-5/trg-5-6.md (hunk 2 above)
Comment (carslen) [reply docs/release/trg-5/trg-5-6.md]: Check

File: docs/release/trg-5/trg-5-6.md (hunk 3 above)
Comment (carslen) [reply docs/release/trg-5/trg-5-6.md]: changed wording

**Pull request merged**